### PR TITLE
Automatically format code using prettier

### PR DESCRIPTION
**Pull request by LostLuma**
beep boop??

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+  "description": "a bot for the r/nfrealmusic discord",
+  "version": "0.1.0",
+  "maintainers": [
+    {
+      "name": "Ahmed",
+      "email": "ahmed@techfromtheheart.com",
+      "web": "https://techfromtheheart.com"
     }
+  ],
+  "homepage": "discord.gg/yzqma4y",
```
Comment (ThaTiemsz) [inline package.json:12]:
```suggestion
  "homepage": "https://discord.gg/yzqma4y",
```

Comment (LostLuma) [reply package.json:12]: Thank for the month late review, friend

Comment (ThaTiemsz) [reply package.json:12]: ## Strings
```diff
# Added
+ SAY_YOU_RE_WELCOME: "You're welcome fren"
```

**Comment by ThaTiemsz** [comment]
## Strings
```diff
# Added
+ SAY_YOU_RE_WELCOME: "You're welcome fren"
```
@LostLuma

**Pull request merged**